services/datamanager/builtin/builtin.go

```diff
@@ -243,7 +247,15 @@ func (svc *builtIn) initializeOrUpdateCollector(
 	if captureBufferSize == 0 {
 		captureBufferSize = defaultCaptureBufferSize
 	}
-
+	additionalParamKey, ok := metadataToAdditionalParamFields[generateMetadataKey(
+		md.MethodMetadata.API.String(),
+		md.MethodMetadata.MethodName)]
+	// only check additional params if it is a board
+	if ok {
+		if _, ok := config.AdditionalParams[additionalParamKey]; !ok {
+			return nil, errors.Errorf("failed to validate additional_params for board, must supply %s", additionalParamKey)
```

Comment:
Rather than specifying board, you can use `md.MethodMetadata.API.String()` (and then remove the 'only check if board' comment above). When this is updated, could you retest manually on the board collectors into app? Thanks Vijay!